src/cli.rs

```diff
@@ -69,4 +69,17 @@ pub enum Commands {
         #[arg(short, long)]
         registry: bool,
     },
+
+    /// Copy an initial rcds.yaml to the current working directory.
```

Comment:
Since this command can do interactive prompts aside from non-interactive copy/write-out, this description should a bit more generic, something like

```suggestion
    /// Set up a new rcds.yaml config in the current working directory
```

Reply:
done